README.adoc

```diff
@@ -761,7 +762,7 @@ description of setup and clean commands below.
 ----
 tebako press \
   -e|--entry-point=<entry-point> \
-  -r|--root=<project-root-folder> \
+  -r|--project-root=<project-root-folder> \

```

Comment:
This is correct --->           `-r|--root=<project-root-folder>`
This is not correct --->           `-r|--project-root=<project-root-folder>`

I would not like to change command line since there are users who have integrated it into their build scripts

Reply:
Agree.